subscription_test.go

```diff
@@ -22,6 +22,7 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"net/http"
```

Comment:
Please group the import together with the other stdlib ones above (respecting alphabetical order).